packages/calcite-components/src/components/date-picker-month-header/date-picker-month-header.e2e.ts

```diff
@@ -1,13 +1,6 @@
 import { newE2EPage } from "@stencil/core/testing";
-import { html } from "../../../support/formatting";
-import { renders } from "../../tests/commonTests";
 import { DateLocaleData } from "../date-picker/utils";
-
 describe("calcite-date-picker-month-header", () => {
-  describe("renders", () => {
```

Comment:
Since `setSelectMenuIconOffset` method invoked in `componentDidLoad`, `render` test util would return build error if component is rendered without `activeDate`.

Reply:
You could use `newProgrammaticE2EPage` to set up the component before rendering (see [example](https://github.com/Esri/calcite-design-system/blob/bcbb79f8c925d505bb4ee5e6a54861c5f6bb88b9/packages/calcite-components/src/components/date-picker-day/date-picker-day.e2e.ts#L7-L23)). `disabled` and other common test helpers support this pattern.